src/utils/fhir.js

```diff
@@ -186,7 +196,25 @@ function subscriptionsFromBundle(specBundle, url, token) {
     throw new Error('Specification Bundle does not contain a PlanDefinition');
   }
 
-  return planDefs.map(planDef => subscriptionsFromPlanDef(planDef.resource, url, token)).flat();
+  const subscriptions = planDefs
+    .map(planDef => subscriptionsFromPlanDef(planDef.resource, url, token))
+    .flat();
+
+  subscriptions.forEach(async subscription => {
+    // Store subscriptions in database
+    debug(`  ...${subscription.resourceType}/${subscription.id}`);
+    db.upsert('subscriptions', subscription, s => s.id === subscription.id);
+
+    // Create/Update Subscriptions on EHR server
```

Comment:
Just to clarify - due to the open question you have on this PR right now this only creates, it doesn't update. I just tested it out and using PUT will handle the create/update for us and it will still handle has a normal subscription (i.e. send notifications correctly. I think the easiest solution then is to just make this `axios.put` to `{ehr}/Subscription/{subscription id}`. For HAPI, we need to make sure the id starts with a letter so if we go this route I would change line 240 to be 
```
const id = `sub${planDef.id}${namedEventCoding.code}`;
```

Reply:
I like this approach too instead of trying to keep track multiple ids. Updated the code to use this!